--- gnet-1.1.4/configure.ac~	Fri May 17 10:14:08 2002
+++ gnet-1.1.4/configure.ac	Fri May 17 10:14:08 2002
@@ -102,9 +102,9 @@
 
 ###############################
 # Check for libraries
-AC_CHECK_LIB([socket], [main])
-AC_CHECK_LIB([nsl], [main])
-AC_CHECK_LIB([resolv], [main])
+#AC_CHECK_LIB([socket], [main])
+#AC_CHECK_LIB([nsl], [main])
+#AC_CHECK_LIB([resolv], [main])
 
 # Wacky pthread madness
 pthread_libs=""
